packages/experiments/src/components/Pagination/Pagination.styles.ts

```diff
@@ -48,11 +48,26 @@ export function getStyles(props: IPaginationStyleProps): IPaginationStyles {
         selectors: {
           '&:aria-selected=true': {
             color: palette.blue,
-            fontWeight: 'bold'
+            cursor: 'default',
+            fontWeight: 'bold',
+            textDecoration: 'underline'
           },
           '&[aria-selected=true]': {
             color: palette.blue,
-            fontWeight: 'bold'
+            cursor: 'default',
+            fontWeight: 'bold',
+            textDecoration: 'underline'
+          },
+          '&:hover:aria-selected=true': {
```

Comment:
Same here... both of the above non-bracketed styles seem invalid or at least redundant. Should be able to safely remove.

Reply:
removed

---
In reply to: [241932268](https://github.com/OfficeDev/office-ui-fabric-react/pull/7411#discussion_r241932268) [](ancestors = 241932268)

Reply:
Thank you 👍